src/amp-user-notification(with_backend_service).html

```diff
@@ -0,0 +1,80 @@
+<!--
+  #### Introduction
+
+  Use amp-user-notification to display a dismissable notification to the user.
+  Use this to implement a dialog to notify users about cookies.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <!-- #### Setup -->
+  <!-- Import the amp-user-notification component in the header. -->
+  <script async custom-element="amp-user-notification" src="https://cdn.ampproject.org/v0/amp-user-notification-0.1.js"></script>
+  <!--
+    Specify any additional css, default css class will put the amp-user-notification element
+    in the left bottom corner.
+  -->
+  <style amp-custom>
+    footer {
+      margin-bottom: 50px;
+    }
+    amp-user-notification {
+      min-height: 30px;
+      font-family: 'Roboto';
+      font-weight: 500;
+      line-height: 30px;
+      padding: 8px;
+      background: #46b6ac;
+    }
+    amp-user-notification button {
+      border: none;
+      border-radius: 2px;
+      color: #fafafa;
+      height: 26px;
+      min-width: 32px;
+      padding: 0 16px;
+      margin: 0 16px;
+      text-transform: uppercase;
+      letter-spacing: 0;
+      cursor: pointer;
+      vertical-align: middle;
+      line-height: 26px;
+      text-align: center;
+      background: #3f51b5;
+    }
+  </style>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Using Backend service -->
+  <!--
+    When using a backend solution, 2 parameters are available: data-show-if-href and
```

Comment:
replace: '2' with 'two'
mark parameters as code: `data-show-if-href` and `data-dismiss-if-href`